modules/constants.js

```diff
@@ -263,3 +267,20 @@ exports.ERROR_TYPE = {
     RPC_INITIALIZATION_ERROR: 'RpcInitializationError',
     KEEP_ALIVE_ERROR: 'KeepAliveError',
 };
+/**
+ * @constant {object} HANDLER_ID_STATUS -
+ *  Possible statuses for handler id
+ */
+exports.HANDLER_ID_STATUS = {
```

Comment:
Should we also have Finished or successful handler id status?

Reply:
These constants are added for new controller, and it's still not finished, once we fully use it we will add all statuses